src/background/services/openPayments.ts

```diff
@@ -459,18 +483,17 @@ export class OpenPaymentsService {
   }
 
   async disconnectWallet() {
-    const { grant } = await this.storage.get(['grant'])
+    const { recurringGrant, oneTimeGrant } = await this.storage.get([
+      'recurringGrant',
+      'oneTimeGrant'
+    ])
+    const grant = recurringGrant || oneTimeGrant
```

Comment:
We should try to revoke both grants of wallet disconnect. For example:
- User sets a recurring amount when connecting the wallet
- Runs out of funds and adds a one-time budget
- User disconnects the wallet.

In this case, we will only revoke the recurring grant, but not the one-time one.

Reply:
This is something I'm leaving for the other PR, when we'd have logic like switchGrant and "top up" one time over existing grant. As of this PR, we'd only have one active grant at a time.

Reply:
> This is something I'm leaving for the other PR, when we'd have logic like switchGrant and "top up" one time over existing grant. As of this PR, we'd only have one active grant at a time.

Add a comment for it.